erpnext/accounts/doctype/tax_withholding_category/tax_withholding_category.py

```diff
@@ -323,25 +323,31 @@ def get_invoice_vouchers(parties, tax_details, company, party_type="Supplier"):
 	for d in invoices_details:
 		vouchers.append(d.name)
 		voucher_wise_amount.update({d.name: {"amount": d.base_net_total, "voucher_type": doctype}})
+	
+	cond = ''
```

Comment:
@ChandanaSPthirvusoft Can you rewrite this query using query builder?

Reply:
Updated.. Kindly verify @deepeshgarg007